### PR TITLE
ln: add Hindi translation

### PR DESCRIPTION
**Pull request by 2KAbhishek**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.hi/common/ln.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ln
+
+> फाइलों और निर्देशिकाओं के लिंक बनाता है।
+> अधिक जानकारी: <https://www.gnu.org/software/coreutils/ln>।
```
Comment (blueskyson) [inline pages.hi/common/ln.md:4]:
```suggestion
> अधिक जानकारी: <https://www.gnu.org/software/coreutils/ln>.
```

File: pages.hi/common/ln.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- किसी फ़ाइल या निर्देशिका के लिए एक प्रतीकात्मक लिंक बनाएँ:
+
+`ln -s {{फ़ाइल या निर्देशिका / का / पथ}} {{लिंक / का / पथ}}`
```
Comment (blueskyson) [inline pages.hi/common/ln.md:8]: I'm not sure whether to put spaces between Hindi characters and slashes. In [chsh](https://github.com/tldr-pages/tldr/blob/main/pages.hi/common/chsh.md), the author didn't use spaces, please let us know which is the correct way in Hindi. In tldr, we tend to not put spaces near slashes.

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution. Here are some suggestions.

File: pages.hi/common/ln.md (hunk 1 above)
Comment (2KAbhishek) [reply pages.hi/common/ln.md:4]: '।' is actually the full stop alternative in hindi, are we sure that we want to use '.' here?

File: pages.hi/common/ln.md (hunk 2 above)
Comment (2KAbhishek) [reply pages.hi/common/ln.md]: I have removed the spaces, they could be confusing, thanks for the suggestion

File: pages.hi/common/ln.md (hunk 1 above)
Comment (marchersimon) [reply pages.hi/common/ln.md]: We should keep what's correct. So it should be `!`.

File: pages.hi/common/ln.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# ln
+
+> फाइलों और निर्देशिकाओं के लिंक बनाता है।
+> अधिक जानकारी: <https://www.gnu.org/software/coreutils/ln>।
+
+- किसी फ़ाइल या निर्देशिका के लिए एक प्रतीकात्मक लिंक बनाएँ:
+
+`ln -s {{फ़ाइल_या_निर्देशिका/का/पथ}} {{लिंक/का/पथ}}`
```
Comment (marchersimon) [inline pages.hi/common/ln.md:8]: This should be an absolute path (same for the other two examples)
```suggestion
`ln -s {{/फ़ाइल_या_निर्देशिका/का/पथ}} {{लिंक/का/पथ}}`
```

**Review by marchersimon — APPROVED** [review]
Thanks @2KAbhishek

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**